auxiliary/eus-character.l

```diff
@@ -0,0 +1,28 @@
+(in-package "LISP")
+
+(defun characterp (ch)
+  "Returns T if ch is a character, false otherwise."
+  (if (or (floatp ch) (derivedp ch object) (< ch 0))
+      nil
+      t))
+
+(defun char-int (ch) (assert (characterp ch)) ch)
+(defun char-not-equal (x y)
+  (assert (and (characterp x) (characterp y)))
+  (not (equal x y)))
+
+(setf (symbol-function 'char<) #'<
```

Comment:
or we have `lisp::char<`

Reply:
yep, this is just the simplest implementation to avoid `function not defined` errors.

We should add real functions later (maybe add character class in `cl-compatible`?)